src/main/scala/tech/cryptonomic/conseil/tezos/michelson/parser/JsonParser.scala

```diff
@@ -55,11 +55,16 @@ object JsonParser {
    * Empty expression is represented as an empty array in JSON.
    *
    * */
-  case class JsonType(prim: String, args: Option[List[Either[JsonExpression, Nil.type]]]) extends JsonExpression {
-    override def toMichelsonExpression = MichelsonType(prim, args.getOrElse(List.empty).map {
-      case Left(it) => it.toMichelsonExpression
-      case Right(_) => MichelsonEmptyExpression
-    })
+  case class JsonType(prim: String,
```

Comment:
Just for the sake of consistency (which is not always there anyway, I know... 😝 ), can we agree to start args on a separate line when they're not all inlined?

i.e.
```scala
case class JsonType(
  prim: String,
  args: ...
  ...
)
```